src/library.js

```diff
@@ -3492,8 +3492,7 @@ mergeInto(LibraryManager.library, {
     '$maybeExit',
 #endif
   ],
-  $callUserCallback__docs: '/** @param {boolean=} synchronous */',
-  $callUserCallback: function(func, synchronous) {
+  $callUserCallback: function(func) {
```

Comment:
the doc comment above should probably mention that this callback is now always asynchronous

Reply:
Why do you think we need extra docs?   I'm pretty sure nobody except fetch as ever using this.. and the synchronous argument was never added to the comment in the first place.

Reply:
Are you refering the doc in this file, or the `__docs` which get placed into the output file (purely for closures benefit).

Reply:
oh I was just referring to the big comment on line 3479-3485 above.

Reply:
I think because that comment never mentioned anything about the synchronous argument there is no need to mention its removal either.